AppCenter/AppCenter/MSACAppCenter.h

```diff
@@ -68,76 +69,45 @@
  *
  * @param services Array of services to start.
  */
-+ (void)startFromLibraryWithServices:(NSArray<Class> *)services;
++ (void)startFromLibraryWithServices:(NSArray<Class> *)services NS_SWIFT_NAME(startFromLibrary(services:));
 
 /**
- * Check whether the SDK has already been configured or not.
- *
- * @return YES if configured, NO otherwise.
+ * The flag indicates whether the SDK has already been configured or not.
  */
-+ (BOOL)isConfigured;
+@property(class, atomic, readonly) BOOL isConfigured;
 
 /**
- * Check whether app is running in App Center Test Cloud.
- *
- * @return true if running in App Center Test Cloud, false otherwise.
+ * The flag indicates whether app is running in App Center Test Cloud.
  */
-+ (BOOL)isRunningInAppCenterTestCloud;
+@property(class, atomic, readonly) BOOL isRunningInAppCenterTestCloud;
 
 /**
- * Change the base URL (schema + authority + port only) used to communicate with the backend.
+ * The flag indicates whether or not the SDK was enabled as a whole
  *
- * @param logUrl Base URL to use for backend communication.
- */
-+ (void)setLogUrl:(NSString *)logUrl;
-
-/**
- * Enable or disable the SDK as a whole. In addition to AppCenter resources, it will also enable or disable all registered services.
  * The state is persisted in the device's storage across application launches.
- *
- * @param isEnabled YES to enable, NO to disable.
- *
- * @see isEnabled
- */
-+ (void)setEnabled:(BOOL)isEnabled;
-
-/**
- * Check whether the SDK is enabled or not as a whole.
- *
- * @return YES if enabled, NO otherwise.
- *
- * @see setEnabled:
  */
-+ (BOOL)isEnabled;
+@property(class, nonatomic, getter=isEnabled, setter=setEnabled:) BOOL enabled NS_SWIFT_NAME(enabled);
```

Comment:
👍 Right, we don't need the `is` prefix in the Swift property because it's mutable.